.github/ISSUE_TEMPLATE/language-request.md

```diff
@@ -0,0 +1,16 @@
+---
+name: Language request
+about: Request adding not existed language to Crowdin
+labels: crowdin,language request
```

Comment:
Do these labels already exist on the repo?

Reply:
Yes, it is